### PR TITLE
add espn ff_standings (#164)

### PR DESCRIPTION
**Pull request by tonyelhabr**

### CODE REVIEW

File: tests/testthat/test-ff_standings.R [hunk 1]
```diff
@@ -18,9 +18,13 @@ with_mock_api({
     got_schedule <- ff_schedule(got_conn, week = 4)
     got_potentialpoints <- .flea_add_potentialpoints(got_schedule, got_conn)
 
+    tony_conn <- espn_connect(season = 2020, league_id = 899513)
+    tony_standings <- ff_standings(tony_conn)
```
Comment (tanho63) [inline tests/testthat/test-ff_standings.R:22]: Do these tests run/work? :)

Comment (tonyelhabr) [reply tests/testthat/test-ff_standings.R:22]: so, i ran `devtools::check()` and was getting an issue with the sleeper `ff_standings()` call, unless i was misreading the output and it was really an issue with this new ESPN `ff_standings()`. but when i ran `testthat::test_local()` every test passed

Comment (tanho63) [reply tests/testthat/test-ff_standings.R:22]: Looking at it rn, brb

Comment (tanho63) [reply tests/testthat/test-ff_standings.R:22]: LGTM! Standings should theoretically incorporate potential points (aka optimal lineups) to be consistent with other methods, I do have some code for it that we could probably replicate once ff_schedule code is finalized

**Comment by tanho63** [comment]
LGTM! I'd like to include something similar to this function for potential points/optimal lineup calculations, eventually... https://github.com/dynastyprocess/apps/blob/3c25bd94756589cb2d9ca0b33c6f1b81caa630c0/potentialpoints/functions.R#L197

File: tests/testthat/test-ff_standings.R (hunk 1 above)
Comment (tonyelhabr) [reply tests/testthat/test-ff_standings.R:22]: oh, so is that is what the potential points are for lol... yeah we can do that. just wanted to get the meat of the function done

Comment (tanho63) [reply tests/testthat/test-ff_standings.R:22]: Can you rerun the test output on devtools check and screenshot the bug to me?

Comment (tonyelhabr) [reply tests/testthat/test-ff_standings.R:22]: i said sleeper earlier but it was actually fleaflicker

Capture

**Pull request merged**